### PR TITLE
Enable cppad on Windows

### PR DESCRIPTION
**Pull request by traversaro**
In particular, check if https://github.com/coin-or/CppAD/issues/60 is fixed.

### CODE REVIEW

**Comment by traversaro** [comment]
There is a unrelated conda/macOS failure (it should be fixed by https://github.com/conda-forge/gazebo-feedstock/pull/38) but other then that the fix seems to be working fine, if we update cppad to the latest version.

File: CMakeLists.txt [hunk 1]
```diff
@@ -169,18 +169,6 @@ if(ROBOTOLOGY_ENABLE_DYNAMICS)
   endif()
 endif()
 
-if(ROBOTOLOGY_ENABLE_DYNAMICS_FULL_DEPS)
-  find_or_build_package(manif)
-  find_or_build_package(qhull)
-  find_or_build_package(casadi)
-
```
Comment (GiulioRomualdi) [inline CMakeLists.txt:176]: This part shouldn't be removed right?

Comment (traversaro) [reply CMakeLists.txt:176]: Sorry, I forgot to describe this in the PR. As we are including these as dependencies of blf, there is no need to also duplicate these in the main CMakeLists.txt . I realized this when I had two modified in two places to enable cppad on Windows.

**Comment by GiulioRomualdi** [comment]
On `macOS` s failing but since it does not depend on this PR I think you can merge it, right?

**Comment by traversaro** [comment]
> 
> 
> On `macOS` s failing but since it does not depend on this PR I think you can merge it, right?

Yes, macOS/Conda failed but it was fixed by https://github.com/conda-forge/gazebo-feedstock/pull/38 .

**Review by GiulioRomualdi — APPROVED** [review]

**Pull request merged**